estate/models/estate_property.py

```diff
@@ -0,0 +1,87 @@
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+
+    name = fields.Char()
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(default=lambda self: fields.Date.today() + relativedelta(months=+3))
```

Comment:
You may also use the method defined on the field type Date:

```suggestion
    date_availability = fields.Date(default=lambda self: fields.Date.add(fields.Date.today(), months=3))
```